core/fields/Field_Select.php

```diff
@@ -44,7 +44,7 @@ public function display() {
 		echo "<div class='field {$hidden} {$required}'>";
 			echo "<label class='label'>" . $this->label . "</label>";
 			echo "<div class='control'>";
-				echo "<div class='" . ($this->slimselect ? "slimselect_select" : "select") . "'>";
+				echo "<div class='" . ($this->slimselect ? "slimselect_select" : "select") . ($this->multiple ? " is-multiple" : false) . "'>";
```

Comment:
should by in the ternary else, since not needed if slimselect is enabled

Reply:
addressed in https://github.com/HoltBosse/Alba/pull/455/commits/d41fbb6e7af6de13f8e9073638d69e7f81f5df17